frontend/src/layouts/MySchedules/MySchedules.tsx

```diff
@@ -1,12 +1,112 @@
+
+  const fetchSchedules = async () => {
+    try {
+      const response = await servicesService.listMySchedules();
+      if (response.schedules && Array.isArray(response.schedules)) {
+        setSchedules(response.schedules);
+      } else {
+        console.error("Formato inesperado da resposta:", response);
+        setSchedules([]);
+      }
+    } catch (error) {
+      console.error("Erro ao buscar agendamentos:", error);
+      setSchedules([]);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    fetchSchedules();
+  }, []);
+
+  const handleScheduleApproval = async (scheduleId: number, approved: boolean) => {
+    try {
+      await servicesService.approveDisapproveSchedule(scheduleId, approved);
+      alert(`Agendamento ${approved ? "aceito" : "rejeitado"} com sucesso!`);
+      fetchSchedules();
+    } catch (error) {
+      console.error("Erro ao atualizar o status do agendamento:", error);
+      alert("Não foi possível atualizar o status do agendamento. Tente novamente.");
+    }
+  };
+
+  const columns = [
+    { Header: "ID", accessor: "id", width: "5%" },
+    { Header: "ID Serviço", accessor: "idServico", width: "5%" },
+    { Header: "ID Usuário", accessor: "idUsuario", width: "5%" },
+    { Header: "Data", accessor: "data", width: "16.5%" },
+    { Header: "Observações", accessor: "observacoes", width: "16.5%" },
+    { Header: "Status", accessor: "status", width: "10%" },
+    { Header: "Ações", accessor: "acoes", width: "20%" },
+  ];
+
+  const rows = schedules.map((schedule) => {
+    console.log(schedule.id, schedule.approved);
+    return {
+      id: schedule.id,
+      idServico: schedule.serviceId,
+      idUsuario: schedule.schedulerUserId,
+      data: new Date(schedule.appointmentDate).toLocaleString(),
+      observacoes: schedule.obs?.trim() || "Nenhuma observação",
+      status:
+        schedule.approved === true
+          ? "Aceito"
+          : schedule.approved === false
+          ? "Desaprovado"
+          : "Pendente",
+      acoes: (
+        <>
+          <MDButton
+            color="success"
+            onClick={() => handleScheduleApproval(schedule.id, true)}
+            style={{
+              marginRight: "10px",
+              padding: "5px 10px",
+            }}
+          >
+            Aceitar
+          </MDButton>
+          <MDButton
+            color="error"
+            onClick={() => handleScheduleApproval(schedule.id, false)}
+            style={{
+              padding: "5px 10px",
+            }}
+          >
+            Desaprovar
+          </MDButton>
+        </>
+      ),
+    };
+  });
+
   return (
     <DashboardLayout>
-      <DashboardNavbar titleToBradcrumb="Serviços" title="Contratar Serviço" />
-      <MDBox>Tela...</MDBox>
+      <DashboardNavbar titleToBradcrumb="Serviços" title="Meus Agendamentos" />
+      <MDBox>
+        {isLoading ? (
+          <p>Carregando agendamentos...</p>
+        ) : (
+          <DataTable
```

Comment:
A tabela está com campos em inglês, altere todos os labels para português. De uma olhada na documentação. Retire o "entries per page" dessa tabela, o back não foi preparado pra receber ele. Se não conseguir alterar para português o "Showing 1 to 10 of 2 entries", então apenas remova-o. [Documentação da tabela](https://www.creative-tim.com/learning-lab/react/data-table/material-dashboard/).